index.js

```diff
@@ -45,9 +46,12 @@ const startGameHandlers = Alexa.CreateStateHandler(states.STARTMODE, {
     this.handler.state = states.GUESSMODE;
     const store = engine.init(this.attributes.gameState);
     store.dispatch(engine.startGame());
-    this.attribute.gameState = store.getState();
-    console.log(this.attribute.gameState);
-    const choices = this.attribute.gameState.offenseCardChoices.playerCards.join(' or ');
+    this.attributes.gameState = store.getState();
+    console.log(store.getState());
```

Comment:
Yeah, you can see it in the lambda logs which in AWS are under CloudWatch > Logs